core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -14,7 +14,7 @@ <h3 class="e2e-test-joyride-title">Preview</h3>
     <div class="navbar-icons">
       <li [ngClass]="{'navbar-tab-active': getActiveTabName() === 'main', 'uib-dropdown': countWarnings()}" (click)="selectMainTab()"
           class="nav-item icon nav-list-item position-relative">
-        <a href="#" class="nav-link navbar-tab e2e-test-main-tab" [ngbTooltip]="'Editor'" placement="{{countWarnings() ? 'left' : 'bottom'}}">
+        <a href="#" class="nav-link navbar-tab e2e-test-main-tab" [ngbTooltip]="'Editor'" placement="{{countWarnings() ? 'left' : 'bottom'}}" aria-label="Exploration Editor Button" >
```

Comment:
```suggestion
        <a href="#"
             class="nav-link navbar-tab e2e-test-main-tab"
             [ngbTooltip]="'Editor'"
             placement="{{countWarnings() ? 'left' : 'bottom'}}"
             aria-label="Exploration Editor Button">
```

Reply:
Ditto elsewhere

Reply:
Done @Rijuta-s

Reply:
Once you d o the changes please unassign yourself and assign the reviewer again! Thanks..